### ISSUE TITLE
Make a Visual Mark on Test Results

### ISSUE THREAD

**Issue by feinstein** [nunit/nunit-vs-adapter#103]
The native Visual Studio Tests have a check mark on everything related to that test, for example:

![captura_de_tela_031616_041224_am](https://cloud.githubusercontent.com/assets/8120472/13804661/9e986400-eb2d-11e5-8c05-80b3c186ee56.jpg)

I don't know if it's possible to mimic this functionality, but it will be a nice addition.

**Comment by OsirisTerje** [comment]
You mean like this ?
![image](https://cloud.githubusercontent.com/assets/203432/13827373/6989aa00-ebbc-11e5-91b3-f09da963bcbe.png)

**Comment by feinstein** [comment]
With the `Test` attribute it works....but not with the `TestCase` attribute:

![captura_de_tela_032416_053905_pm](https://cloud.githubusercontent.com/assets/8120472/14030672/a514c6be-f1e7-11e5-9943-003e82d2a3be.jpg)

Even though the Test Explorer does show the completed successful tests with a check mark:

![captura_de_tela_032416_054043_pm](https://cloud.githubusercontent.com/assets/8120472/14030714/e2d64338-f1e7-11e5-9287-8d47aa00af27.jpg)

**Comment by CharliePoole** [comment]
Does the explorer window show the correct file location for the test method? If not, there may be something we can fix. But if it does, that means we are passing the correct info to VS and it's basically up to them to do something with it. :-(

**Comment by feinstein** [comment]
Yes, I think it is showing the right file location...can you make a simple test like mine and see if this isn't something isolated?

**Comment by CharliePoole** [comment]
I have verified this behavior using the NUnit 3 adapter rather than NUnit 2, which seems to indicate it's just how it works with VS.

The behavior makes a certain amount of sense. If a method is parameterized and has twenty cases, potentially having differing results, then what would be shown?

I'd like to close this as notabug if nobody objects.

**Comment by feinstein** [comment]
Well, I think if it has 20 cases, and all of them pass, so there should be a green check mark...if one fails, even if the others pass, there should be a red warning....Just like it is in the Test Explorer.

**Comment by CharliePoole** [comment]
Perhaps, but definitely not in our control.

**Comment by feinstein** [comment]
I see....do you have a channel of communication with Microsoft to raise this issue with them? I think this will be a nice addition.

**Comment by CharliePoole** [comment]
@OsirisTerje can you try? They don't talk to me for sure.

**Comment by OsirisTerje** [comment]
I'll talk with them

**Comment by CharliePoole** [comment]
Thanks @OsirisTerje 

If this turns out to be something they don't expect to change anytime soon, I think we should just close it as not our bug.
